pages/settings/teams.tsx

```diff
@@ -153,4 +166,16 @@ export default function Teams(props) {
       </SettingsShell>
     </Shell>
   );
-}
\ No newline at end of file
+}
+
+// Export the `session` prop to use sessions with Server Side Rendering
+export const getServerSideProps: GetServerSideProps<{ session: Session | null }> = async (context) => {
+    const session = await getSession(context);
+    if (!session) {
+      return { redirect: { permanent: false, destination: '/auth/login' } };
+    }
+
+    return {
+      props: { session }
```

Comment:
Codacy has a fix for the issue: [Replace `······props:·{·session·}` with `····props:·{·session·},`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
    props: { session },
```